src/views/search/search.jsx

```diff
@@ -73,7 +100,8 @@ class Search extends React.Component {
         const locale = this.props.intl.locale;
         const loadNumber = this.state.loadNumber;
         const offset = this.state.offset;
-        const queryString = `limit=${loadNumber}&offset=${offset}&language=${locale}&mode=popular${termText}`;
+        const mode = this.state.mode;
```

Comment:
Since you're already referring to `this.state.mode` here, the change from Redux state to local React state should work.